src/java.base/share/classes/com/sun/crypto/provider/FeedbackCipher.java

```diff
@@ -140,6 +140,11 @@ abstract void init(boolean decrypting, String algorithm, byte[] key,
      */
     abstract int encrypt(byte[] plain, int plainOffset, int plainLen,
                          byte[] cipher, int cipherOffset);
+
+    int encrypt(byte[] plain, int plainOffset, int length) {
```

Comment:
Is it really necessary to modify this class and add this?

Reply:
That was necessary at some point, however I tried it now and it seems to compile fine without it. Hence, I removed it.